fedora-coreos/provisioning/fcos-lab-tutorial.bu

```diff
@@ -62,24 +72,28 @@ storage:
     - path: /usr/local/bin/butane
       mode: 0755
       contents:
-        source: https://github.com/coreos/butane/releases/download/v0.18.0/butane-x86_64-unknown-linux-gnu
+        source: https://github.com/coreos/butane/releases/download/v0.20.0/butane-x86_64-unknown-linux-gnu
     - path: /usr/local/bin/ignition-validate
       mode: 0755
       contents:
-        source: https://github.com/coreos/ignition/releases/download/v2.16.2/ignition-validate-x86_64-linux
+        source: https://github.com/coreos/ignition/releases/download/v2.18.0/ignition-validate-x86_64-linux
+    - path: /usr/local/bin/launch_coreos_vm
+      mode: 0755
+      contents:
+        source: https://raw.githubusercontent.com/jbtrystram/devconf24-coreos-workshop/main/launch_coreos_vm
```

Comment:
Let's push this to this repo?

Reply:
Done. It was overwritten by the force push